app/views/branch_crud/index.html.erb

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+  <html>
+  <head>
+  <script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-fQybjgWLrvvRgtW6bFlB7jaZrFsaBXjsOMm/tB9LTS58ONXgqbR9W8oWht/amnpF" crossorigin="anonymous"></script>
+  </head>
+  <body>
```

Comment:
This is not needed at all, rails views only load the content while the application.html.erb file does content the "layout", loads the styles and the js/scripts 😄 

Consider taking a look to this resource https://guides.rubyonrails.org/layouts_and_rendering.html